versions/v1.0.0/Social/Share/main.adoc

```diff
@@ -39,6 +39,35 @@ interface Share {
 }
 ----
 
+===== Fields
+
+`handle` *Optional*:: Handle/username of the user who is sharing, e.g. `rabby1.lens`.
+`body` *Optional*:: Main content of the post.
+`title` *Optional*:: Title of the post.
+`summary` *Optional*:: Short summary of the post.
+`media` *Optional*:: Array of media objects that are attached to the post.
+`media.address` *Required*:: Url/address of the media object.
+`media.mime_type` *Required*:: Mime type of the media object.
```

Comment:
Is there a need to explain the meaning of mime type? What is it?

Reply:
a specific standard for media types, may be no need to explain in a protocol doc?

https://developer.mozilla.org/en-US/docs/Glossary/MIME_type